blockprotocol/blockprotocol
# [SITE] Add preview images for blocks

This PR 
- updates the codebase to pull block preview images from the block's metadata file
- adds some styling improvements to the block card and block slider components.

How to test this
- Change the `branch` property in the registry files (`hub/*.json`)  to `vu/block-preview-images`
- Run `yarn build-blocks` from the `site/` directory (might need to delete the `public/blocks` folder before doing this)
- Run `yarn dev`
- The preview images should show up when you go to the hub page ("/hub")

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201555682469130
